packages/simcore-sdk/src/simcore_sdk/node_ports_common/file_io_utils.py

```diff
@@ -289,7 +293,7 @@ async def _upload_file_part(
             ) as response:
                 await _raise_for_status(response)
                 if io_log_redirect_cb and pbar.update(file_part_size):
-                    with log_catch(log, reraise=False):
+                    with log_catch(_logger, reraise=False):
```

Comment:
will not fix

Reply:
> :-D let me remind you the words of a wise man who told me either we use it or not ... but if we use it, we have to respect the "ruling" ...
> Since sonarcloud is doing a great job i suggest to deactivate codeclimate

Can I make a PR to kill it? 🔥

Reply:
Now seriously. I'd like to be able to skip this, since refactoring the function, will make it read worse.